src/lib.rs

```diff
@@ -891,6 +895,12 @@ impl<'a> WebViewBuilder<'a> {
     self
   }
 
+  /// Whether JavaScript code should be injected to all subframes
```

Comment:
I think this option will need to be more granular, i.e. choose which script should be injected into subframes and which is not. cc @tweidinger

Reply:
Instead of adding a new field, you can just modify [`initialization_scripts`](https://github.com/Norbiros/wry/blob/c65ca8486966a996f640aa0c2510ee1bf6b07c3e/src/lib.rs#L358) field and change its type from `Vec<String>` to `Vec<(String, bool)>` where the bool defines whether to inject into subframes or not